pkg/build/controller/build/defaults/api/v1/types.go

```diff
@@ -43,6 +43,10 @@ type BuildDefaultsConfig struct {
 
 	// resources defines resource requirements to execute the build.
 	Resources kapi.ResourceRequirements `json:"resources,omitempty"`
+
+	// tolerations is a list of default Tolerations that are applied to build pods
+	// so that they can be placed on build-specific nodes
+	Tolerations []kapi.Toleration `json:"tolerations,omitempty"`
```

Comment:
@deads2k should this whole file not have moved into the api repo?

Reply:
> @deads2k should this whole file not have moved into the api repo?

The admission ones need to move. Found it late last month.  Is this still configured via admission?  Try to fix that in 3.9, so we can fix the config in 3.10.